cs-cz/sass.html.markdown

```diff
@@ -0,0 +1,439 @@
+---
+language: sass
+filename: learnsass.scss
+contributors:
+  - ["Laura Kyle", "https://github.com/LauraNK"]
+  - ["Sean Corrales", "https://github.com/droidenator"]
+translators:
+  - ["Michal Martinek", "https://github.com/MichalMartinek"]
+
```

Comment:
Could you add a `lang` line? I know it's a bit weird, but the term "language" gets a bit overloaded these days.

```
lang: cs-cz
```